internal/operands/template-validator/resources.go

```diff
@@ -60,7 +56,9 @@ func newServiceAccount(namespace string) *core.ServiceAccount {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      ServiceAccountName,
 			Namespace: namespace,
-			Labels:    commonLabels,
```

Comment:
The changes in this file should cause more duplication than before.
What's the reason/goal of this change? Reducing global vars?

Reply:
Yes, the goal is to remove global variables, because they are mutable and shared between created resources.
Do you remember the problem caused by global variable in your PR: #92 ?

Reply:
Right, I like that idea not a fan of global vars.
Just would also like to try and avoid duplication like this.